tock/hours/views.py

```diff
@@ -583,8 +578,12 @@ def prefilled_formset(self):
             for project_id, hours in timecard_prefills.items()
         ]
 
-        formset = timecard_formset_factory(extra=extra)
-        return formset(initial=init)
+        # Render all of our initial forms and an extra blank one
+        extra = len(init) + 1
+
+        FormSet = timecard_formset_factory(extra=extra)
```

Comment:
python ignorance question: what's the repercussion (if any?) of setting this is a class rather than just a variable?

Reply:
Good question!

Functionally, thanks python, there's no difference between us calling this `FormSet` or `formset`. 

I changed to `FormSet` here to be a bit more idiomatic knowing a few Tock/Django details:
* we know that `timecard_formset_factory` is a passthru to `inlineformset_factory`
* We know that `inlineformset_factory` and [formset factories in general, return classes](https://docs.djangoproject.com/en/2.2/ref/forms/formsets/#django.forms.formsets.formset_factory)

Reply:
This _might_ help https://docs.djangoproject.com/en/3.1/topics/forms/formsets/ Or not. Cause django class-based-views can get hairy.

Reply:
thank you both!!!